site/search/SearchChartHitDataDisplay.tsx

```diff
@@ -0,0 +1,61 @@
+import { faLocationDot } from "@fortawesome/free-solid-svg-icons"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { GrapherTrendArrow } from "@ourworldindata/components"
+
+export interface SearchChartHitDataDisplayProps {
+    entityName: string
+    endValue: string
+    time: string
+    unit?: string
+    startValue?: string // if given, display as a range
+    trend?: "down" | "right" | "up" // only relevant if startValue is given
+    showLocationIcon?: boolean
+}
+
+export function SearchChartHitDataDisplay({
+    entityName,
+    endValue,
+    time,
+    unit,
+    startValue,
+    trend,
+    showLocationIcon,
+}: SearchChartHitDataDisplayProps): React.ReactElement | null {
+    // Remove parentheses from the beginning and end of the unit
+    const strippedUnit = unit?.replace(/(^\(|\)$)/g, "")
```

Comment:
Too bad we got the parentheses into the data. Maybe we could clean it up (in the DB? ETL?) in the future?